modules/developers-guide/pages/glossary.adoc

```diff
@@ -8,53 +8,45 @@ actor::
 
 [[g-ledger]]
 ledger::
-  The Internet Computer records all transactions involving ICP tokens  in a specialized management canister, called the **ledger canister**.The ledger canister is a simplified parallel blockchain that runs on the Internet Computer in a subnet with other network management canisters.
-  The ledger canister implements a smart contract that holds **accounts** and **balances** and keeps a history of the *transactions* that affect accounts and balances. The transactions are recorded to track the following specific events:
+  The {IC} blockchain records all transactions involving ICP tokens in a specialized management canister smart contract, called the **ledger canister**.
+  The ledger canister implements a smart contract that maintains **accounts** and **balances** and keeps a history of the *transactions* that affect accounts and balances. The transactions are recorded to track the following specific events:
   
   * **Mint ICP tokens** for accounts.
   * **Transfer ICP tokens** from one account to another.
   * **Burn ICP tokens** to eliminate them from existence.
 
+[[g-wallet]]
+wallet::
+  A wallet is a specialized dapp (implemented as a canister smart contract) that allows to store and manage cycles. Cycles are used to pay for computation and resource consumption on the {IC} blockchain. Cycles can be obtained by conversion from the {IC} blockchain's native utility token ICP.
+
 [[g-node]]
 node::
-  A physical computer that is a registered member of the {platform} network and running the {IC} replica processes.
-
-[[g-principal]]
-principal::
-  The first time you use the {sdk-short-name}, the `+dfx+` command-line tool creates a `+default+` developer identity for you with a public/private key pair in a PEM file. This {IC} developer identity is represented internally by a derived **principal** data type and a textual representation of the principal often referred to as your **principal identifier**.
-  The developer identity can also be used to derive an **account identifier**—similar to a Bitcoin or Ethereum address—to hold ICP tokens on your behalf in the {IC} ledger canister.
+  A physical or virtual machine that hosts all the hardware, replica software, and configuration settings required to participate in the {IC} blockchain.
 
 [[g-replica]]
 replica:: 
-  In the context of the {platform}, a replica refers to the {IC} processes (for example, the `+replica+`, `+nodemanager+`, and other lower-level {IC} protocol processes) running on a physical computer node in the network.
+  In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes (for example, the `+replica+`, `+nodemanager+`) running on a node.
   For the {sdk-short-name}, you use the `+dfx start+` and `+dfx stop+` commands to start and stop the `+replica+` process locally to provide a local network for development.
```

Comment:
Is there now one or several processes that a make up a replica and which are started and stopped with dfx?

Reply:
Great point this is still somewhat confusing. How about this?
```
In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes running on a node.
  For the {sdk-short-name}, you can use the `+dfx start+` and `+dfx stop+` commands to start and stop a `+replica+` process that provides a local network for development.
```

Reply:
Or maybe even something like this
```
In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes running on a node. For the {sdk-short-name}, you can use the `+dfx start+` and `+dfx stop+` commands to start and stop a `+replica+` process that emulates a local network for development purposes.
```

Reply:
I like the second one much better, but I'd use processes in plural in the second one as well (unless it's only one?)

Reply:
Good point. Done here 4fa6444.